packages/core/src/external/surescripts/client.ts

```diff
@@ -318,14 +320,41 @@ export class SurescriptsSftpClient extends SftpClient {
    * @param requester the requester data for Surescripts data
    * @throws an error if the requester's NPI is invalid
    */
-  private validateRequester(requester: SurescriptsRequesterData): void {
+  private async validateRequester(requester: SurescriptsRequesterData): Promise<void> {
     if (!validateNPI(requester.facility.npi)) {
       this.log(`Invalid NPI "${requester.facility.npi}" for CX ID "${requester.cxId}"`);
       throw new MetriportError("Invalid NPI", undefined, {
         npiNumber: requester.facility.npi,
         cxId: requester.cxId,
       });
     }
+    const isSurescriptsEnabled = await isSurescriptsFeatureFlagEnabledForCx(requester.cxId);
+    if (!isSurescriptsEnabled) {
```

Comment:
Wondering here, we don't have an entry point that we can add this check into? Or is it all through script for now?

Reply:
As of right now there isn't an entry point besides this one, but I did add to the DD for V2 that the validation should ideally be farther up stream in an internal API endpoint.